migrations/src/postDeploymentSetup.js

```diff
@@ -76,7 +76,7 @@ module.exports = async () => {
   const stakerAddressList = STAKER_ADDRESSES.split(',');
 
   // Only transfer tokens in testnets
-  if (NETWORK !== 'mainnet') {
+  if (NETWORK === 'testnet') {
```

Comment:
Do we need to define the testnet networks explicitly here?

Reply:
Introduce a new env variable, NETWORK_TYPE. Use that here. It will be either "MAINNET" or "TESTNET".